### ISSUE TITLE
WCAG 2.1 達成方法集のリリース準備（2020-09）

### ISSUE THREAD

**Issue by momdo** [waic/wcag21#637]
- [x] waic.jpのURLを決める
- [ ] 解説書から達成方法集へのURLを置換する
- [x] リリース機能でリリースする
- [x] ステージング環境にアップロードする
- [x] WAICレビューのメールを主査が送信する
- [x] レビュー期間中のコメントに対する必要なアクションがないことを確認する
- [x] 本番環境で公開する
- [x] 必要であればWAICのニュースとして告知する

**Comment by bakera** [comment]
>解説書から達成方法集へのURLを置換する

本家へのリンクになっているので、達成方法集を公開後にリンクを日本語訳に向ける。
達成方法集公開後に実施 (なので後回し)。

**Comment by bakera** [comment]
>waic.jpのURLを決める

解説書のURLが以下なので、
https://waic.jp/docs/WCAG21/Understanding/

達成方法集は以下となる。
https://waic.jp/docs/WCAG21/Techniques/

議論が必要なほどではないので、このURLで公開しますよ、という告知をSlackで行う。

**Comment by bakera** [comment]
リリース作成しました

https://github.com/waic/wcag21/releases/tag/Techniques-202009

**Comment by momdo** [comment]
9/28（月）レビュー依頼メール送信済み、10/12（月）〆

**Comment by momdo** [comment]
レビューコメントはメーリングリスト、GitHubともになし。

**Comment by momdo** [comment]
🎉
https://waic.jp/news/20201027/
